### PR TITLE
feat: add new signin/login page

### PR DESCRIPTION
**Pull request by mamadoudicko**
Issue https://github.com/StanGirard/quivr/issues/1404

https://github.com/StanGirard/quivr/assets/63923024/703fedf3-2f39-407b-afc2-78318829ca0f

### CODE REVIEW

File: frontend/app/(auth)/login/__tests__/page.test.tsx [hunk 1]
```diff
@@ -19,13 +19,17 @@ vi.mock("@/lib/context/SupabaseProvider", () => ({
...
+  QuivrLogo: () => <div>LOGO</div>,
+}));
+
 vi.mock("@/services/analytics/june/useEventTracking", () => ({
   useEventTracking: () => ({ track: vi.fn() }),
 }));
 
 describe("Login component", () => {
   afterEach(() => {
-    vi.clearAllMocks();
```
Comment (gozineb) [inline frontend/app/(auth)/login/__tests__/page.test.tsx:28]: Why the change ?

File: frontend/app/(auth)/login/page.tsx [hunk 2]
```diff
@@ -3,78 +3,49 @@ import Link from "next/link";
...
+            {t("talk_to", { ns: "login" })}{" "}
+            <span className="text-primary">Quivr</span>
+          </p>
+          <div className="mt-5 flex flex-col">
             <Field
               name="email"
-              required
               type="email"
-              placeholder={t("email")}
+              placeholder={"Email address"}
```
Comment (gozineb) [inline frontend/app/(auth)/login/page.tsx:33]: Why was the translation removed ?

**Review by gozineb — APPROVED** [review]

File: frontend/app/(auth)/login/page.tsx (hunk 2 above)
Comment (mamadoudicko) [reply frontend/app/(auth)/login/page.tsx]: 🥶

**Review by gozineb — APPROVED** [review]

File: frontend/app/(auth)/login/__tests__/page.test.tsx (hunk 1 above)
Comment (mamadoudicko) [reply frontend/app/(auth)/login/__tests__/page.test.tsx:28]: I was having an error when running tests in parallel while using `vi.clearAllMocks()`. It seems to have a side effect on other tests
